app/scripts/views/sign_up.js

```diff
@@ -257,6 +258,15 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
       this.navigate('cannot_create_account');
     },
 
+    _isSyncChooser: function () {
+      var chooseSyncCapability = this.broker.getCapabilityByName('choose_what_to_sync');
+      // if this is sync and broker is capable of using choose what to sync on the web v1
+      if (this.relier.isSync() && chooseSyncCapability && chooseSyncCapability.version === 'web_v1') {
+        // even if the client is capable, let able to determine if we want to opt-in the user into this behaviour
+        return this.relier.isChooseWhatToSyncWeb(this.user.get('uniqueUserId'), this.window.location.search, this._able);
```

Comment:
I wonder how much this a/b testing will complicate things for Firefox client code.  IIUC it will have to advertise its ability to accept choose-what-to-sync from the web via capabilities message, but still be prepared to not receive anything back and show the native UI.  May be a good tradeoff though, having able for killswitch and progressive rollout will be valuable.